sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -117,6 +118,12 @@ class SQLContext private[sql](
    *
    * @since 1.6.0
    */
+
+  /** Pre-Load built-in User Defined Types */
```

Comment:
I may not understand your point here.

If we remove this, then we don't pre-register UDT for Vector and Matrix. But I think we need to do that, right?